src/hooks/__tests__/useScorepads.test.ts

```diff
@@ -0,0 +1,48 @@
+import { renderHook } from '@testing-library/react-hooks';
+import useApi from '../useApi';
+import useScorepads from '../useScorepads';
+
+jest.mock('../useApi.ts');
+
+describe('useScorepads', () => {
```

Comment:
You mentioned that you weren't sure how to test the error case here, but I don't think you really need to consider error handling here, since the hook itself doesn't do anything with the error, it just passes it along.
So I think it would be sufficient to just have a test in which the `useApi` mock returns an `error`, and then to make sure that `useScorepads()` also returns said error. So it would be something like:
```
const error = new Error('API Error');
(useApi as jest.Mock).mockReturnValue([error, null]);

const { result } = renderHook(() => useScorepads('JanK'));
expect(result.current[0]).toBe(error);
```